dashboard/test/lib/pd/payment/payment_calculator_counselor_admin_test.rb

```diff
@@ -96,5 +91,23 @@ class PaymentCalculatorCounselorAdminTest < ActiveSupport::TestCase
       assert_equal expected_payment_amounts, payment.amounts
       assert_equal 3162.5, payment.total
     end
+
+    test 'no user account' do
+      workshop = create :pd_ended_workshop,
+        workshop_type: Pd::Workshop::TYPE_PUBLIC,
+        course: Pd::Workshop::COURSE_COUNSELOR,
+        num_sessions: 1
+
+      5.times do
+        enrollment = create :pd_enrollment, workshop: workshop
+        create :pd_attendance, session: workshop.sessions.first, enrollment: enrollment
+      end
+
+      workshop_summary = PaymentCalculatorCounselorAdmin.instance.calculate(workshop)
+
+      assert_equal 5, workshop_summary.num_teachers
+      assert_equal 5, workshop_summary.num_qualified_teachers
+      assert_equal 5, workshop_summary.total_teacher_attendance_days
+    end
```

Comment:
This tests the scenario that was blowing up the report before, when it relied on teacher user accounts instead of enrollments.